.github/workflows/check-vulns.yml

```diff
@@ -1,17 +1,21 @@
-name: Reusable flow to check for vunls in Depenencies of Node.js branch
+name: Check for vulnerabilities in dependencies of a Node.js branch
```

Comment:
@facutuesca I had made this one the "reusable flow" which we could then call from a action for each of the release lines.  That is why it does not have workflow_dispatch. Adding that is ok, provided that does not break it being called/reused from the other actions.

Reply:
> Adding that is ok, provided that does not break it

It should still work as a reusable workflow, since `on workflow_call` is still there. Now it just supports one more mode of execution (manual trigger)

> Based on that I'm thinking we should leave the title the same.

I'll change it back

Reply:
Fixed